jujugui/static/gui/src/app/components/user-profile/_user-profile.scss

```diff
@@ -1,7 +1,67 @@
 .user-profile {
+
     &__create-new {
         float: right;
         font-size: 1rem;
+        width: 400px;
+        text-align: right;
+
+        &.collapsed {
+            .deployment-input__field,
+            .button--inline-neutral.second {
+                width: 0;
+                border-width: 0;
+            }
+            .deployment-input__field {
+                padding: 0;
+            }
+            .button--inline-neutral.first {
+                width: 150px;
+                border-width: 1px;
+            }
+        }
+
+        .button--inline-neutral.first {
+            width: 0;
+            border-width: 0;
+        }
+
+        .button--inline-neutral {
+            overflow: hidden;
+            white-space: nowrap;
+            transition: all 0.5s ease-in;
```

Comment:
This will override the transition on hover. Could to transition only the style you are changing?